notifications.json

```diff
@@ -1,3 +1,11 @@
 {
-  "notifications": []
+  "notifications": [
+          {
+            "title": "Have a feature or a bug?",
+            "body": "Would you like to see specific features added or have a bug to report? <p> <a href="https://github.com/RedisInsight/RedisInsight">Share</a> it with us! <p> And star the repository if you like RedisInsight!",
```

Comment:
This is not a valid json. We have tests failed. Please escape `"` char with `\"` inside strings.
`"Would you like to see specific features added or have a bug to report? <p> <a href="https://github.com/RedisInsight/RedisInsight">Share</a> it with us! <p> And star the repository if you like RedisInsight!",`
should be 
`"Would you like to see specific features added or have a bug to report? <p> <a href=\"https://github.com/RedisInsight/RedisInsight">Share</a> it with us! <p> And star the repository if you like RedisInsight!\"",`

Reply:
UPD: Here is the correct json with `"` escaped
`"Would you like to see specific features added or have a bug to report? <p> <a href=\"https://github.com/RedisInsight/RedisInsight\">Share</a> it with us! <p> And star the repository if you like RedisInsight!"`